portfolio/src/main/webapp/script.js

```diff
@@ -70,3 +70,16 @@ async function deleteComments() {
   const messagesElement = document.getElementById('comments-container');
   messagesElement.innerHTML = '';
 }
+
+function handleSubmit(e) {
+  e.preventDefault();
+  $.ajax({
+    type: 'POST',
+    url: '/data',
+    data: $(this).serialize(),
+    success: () => fetchContent(),
```

Comment:
If you're not passing arguments, you should be able to do a shorthand `success: fetchContent,`